08.ls_object/format.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+class Format
+  def initialize(sorted_current_dir_files)
+    @sorted_current_dir_files = sorted_current_dir_files
```

Comment:
ちょっと変数名が長い感がありますね。何か削っても良い部分はないかどうか考えてみましょう。

例えば、`sorted`の部分。ここには`sorted`でない方の`current_dir_files`とかは出てこないので、なくても良さそうかなと思いました。あと、ソートされてるかどうかはこのクラスを`new `する呼び出し元がちゃんとソート済みの値を引数に入れてくれるかに依ってしまうという懸念もありますね。

次に`current`の部分。`Format`クラスにとってみれば、この変数はあくまでコンストラクタに渡されたものでしかないです。なので、カレントディレクトリなのかどうかというのは正直分からないんですね。このクラスを`new`した呼び出し元がどんな値を入れてくるのかは読めないわけです。もしかしたらカレントディレクトリではなく別のパス(`../`とか)にあるファイル/フォルダ名を渡してくるかもしれないので、そういう場合は`current`という名前はむしろ適切でなくなってしまう可能性もあります。

なので、`Command`クラスの方で`sorted_current_dir_files`という変数名を使うのはいいんですが、`Format`クラスの方ではただ単に`dir_files`という名前にしてもいいのかなという風に思いました。

ソートされているのかどうか、カレントディレクトリにあるファイル/フォルダなのか、そういう情報は実は`Format`クラスにとってはあまり重要じゃないんですね。こいつにとって重要なのは「何かしらのファイル名/フォルダ名である」ということなんです。`sorted`の方はまああってもいいですけど、本当にソートされてるのかは呼び出し元次第なのでなくてもいい。`current`については、カレントディレクトリでないパスのものを扱う可能性もありますので、むしろない方がいいと思います。

長々と書いちゃいましたが、上記のような理由で、こうするのがいいと思うのですがどうでしょう？
```suggestion
    @dir_files = dir_files
```
各メソッドに出てくるローカル変数からも`sorted`と`current`を取り除いてやれば結構変数名が短くなって読みやすくなるような気がします。